### PR TITLE
Feat/bury cremation

### PR DESCRIPTION
**Pull request by ZIggyVeerman**

### CODE REVIEW

File: src/entities/buryCremation.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { ObjectType, Field } from 'type-graphql';
+import { prop as Property } from '@typegoose/typegoose';
+
+enum buryCremation {
```
Comment (FinnBueno) [inline src/entities/buryCremation.ts:4]: Class & enum names are usually CamelCase; BuryCremation.

File: src/resolver/buryCremation/input.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import { InputType, Field } from 'type-graphql';
+
+enum buryCremation {
```
Comment (FinnBueno) [inline src/resolver/buryCremation/input.ts:3]: Why not import the one from the entity?

**Review by FinnBueno — CHANGES_REQUESTED** [review]
Small changes to the enum are required, that's all.

File: src/resolver/buryCremation/input.ts (hunk 2 above)
Comment (ZIggyVeerman) [reply src/resolver/buryCremation/input.ts:3]: true must be my sleepy head

**Review by FinnBueno — CHANGES_REQUESTED** [review]
Small thing left regarding the type of an enum field.

**Pull request merged**